voila/app.py

```diff
@@ -125,6 +138,7 @@ def start(self):
                 'comm_msg',
                 'comm_info_request',
                 'kernel_info_request',
+                'custom_message',
```

Comment:
Do we need custom_message here?

Reply:
I was confused about this, and now even more. I remember seeing warnings in the console from voila that `custom_message` was send and ignored. I assumed that it was a custom message like the Button widget uses to communicate the click event. But that should be a regular `comm_msg` right? So what is custom_message, and who would send that?

Reply:
It is not part of the kernel protocol. I think it is an ad-hoc message that was used by the `ipywidgets_server` to trigger initial execution.

Reply:
That must have been it, so it can be removed.